src/sass/shared/skeleton.scss

```diff
@@ -0,0 +1,15 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+html, body {
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    font-family: $font-source-sans-pro;
```

Comment:
I rather have a variable named about what it is and not about its value in here.

You can have something like:

```scss
$font-family-source-sans-pro: 'Source Sans Pro', sans-serif;
$font-family-default: $font-family-source-sans-pro;
```